app/style/content/conversation/conversation-input.less

```diff
@@ -167,3 +167,27 @@
 .conversation-input-paste-cancel {
   margin-right: 32px;
 }
+
+.conversation-input-emoji-list {
+  z-index: 2147483000;
+  position: fixed;
+  display: block;
+  box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.2);
+  background: white;
+
+  div {
+    border-bottom: 1px solid #eee;
+    padding: 4px 8px;
+
+    &.selected {
+      background: #2391d3;
```

Comment:
Right.. some magic will be needed with the themes here then! But it's a bit too late to figure out what exactly :)